pdr_backend/pdr_dashboard/test/conftest.py

```diff
@@ -60,6 +60,7 @@ def pytest_setup_options():
     options = Options()
     options.add_argument("--headless")
     options.add_argument("--disable-gpu")
+    options.add_argument("--disable-search-engine-choice-screen")
```

Comment:
this is moot and confusing in general CIs. It it's headless, there's no search engine choice screen.
How about leaving some options commented, with an explanatory comment saying "these are useful when debugging"?

Reply:
I don't see why this could have any bad effect on CIs, Imo it's better to have it in case anyone else encounters the issue